reactor-test/src/main/java/reactor/test/DefaultStepVerifierBuilder.java

```diff
@@ -116,10 +113,12 @@ static void checkStrictlyPositive(long n) {
 			@Nullable Supplier<? extends Publisher<? extends T>> sourceSupplier) {
 		this.initialRequest = options.getInitialRequest();
 		this.options = options;
+		this.errorFormatter = new ErrorFormatter(options.getScenarioName());
```

Comment:
Can we optimize for the default scenarioName == null using a constant ErrorFormatter ?